steering-files/src/main/resources/org/hps/steering/recon/PhysicsRun2019_pass0_KFOnly.lcsim

```diff
@@ -87,7 +89,7 @@
        
         <driver name="TrackerHitDriver" type="org.hps.recon.tracking.DataTrackerHitDriver">
             <neighborDeltaT>8.0</neighborDeltaT>
-            <saveMonsterEvents>false</saveMonsterEvents>
+            <saveMonsterEvents>true</saveMonsterEvents>
```

Comment:
Why is this changed from false to true?

Reply:
I don't believe the current way of handling these events is correct, and simply skipping these events may be giving us a false impression of how well we're doing. I'd like to see what effect this has in combination with enabling the event flag filter.

Reply:
The event flag filter is not currently activated in this steering file.

Reply:
If this is your intention, please add it to this steering file and I will approve this PR.